src/pages/console/auth/user-management.mdx

```diff
@@ -33,7 +33,7 @@ A user can be confirmed by using the [pre-built UI components](/lib/auth/getting
 
 ## To create a group
 
-![Groups](/images/studio/auth/user-groups.png)
+![Process to create Amazon Cognito user pool groups](/images/studio/auth/user-groups.png)
```

Comment:
I think this could also be decorative as it doesn't give me any extra info that isn't already in the text underneath

```suggestion
![](/images/studio/auth/user-groups.png)
```